app/scienceapi/settings.py

```diff
@@ -176,3 +179,4 @@
     CORS_ORIGIN_ALLOW_ALL = True
 else:
     CORS_ORIGIN_WHITELIST = env('CORS_WHITELIST')
+    CORS_ORIGIN_REGEX_WHITELIST = env('CORS_REGEX_WHITELIST')
```

Comment:
Yes because some of them are dynamic paths and some we know the exact domain for. Regexing exact domains is very tricky so we use both instead.